ROOTFS/etc/waggle/sanity/fatal/core_rpi.test

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+pf="RPI test"
+echo "$pf: Begin"
+
+# check core for rpi mount first
+if [ "$(df -h | grep "/media/rpi")" = "" ];
+then 
+        echo "$pf: RPI not mounted at /media/rpi FAIL"
+        exit 9
+fi
+
+# make sure nfs-server is running on core
+if [ "$(systemctl status nfs-server.service | grep "active (exited)")" = "" ];
+then
+        echo "$pf: nfs-server not running on core FAIL"
+        exit 10
+fi
+
+if [ "$(systemctl status dnsmasq.service | grep "active (running)")" = "" ]; then
+        echo "$pf: dnsmasq is not running FAIL"
+        exit 11
+fi
+
+# historial fsck
+uptime_timestamp=$(uptime --since)
+
+if [ "$(journalctl --since "$uptime_timestamp" | grep "/dev/mmcblk0p11: clean")" = "" ];
```

Comment:
Looks like a fsck check leaked into the rpi tests, I think the last file logs to dnsmasq with the following:

```
sent /media/rpi/sage-utils/dhcp-pxe/tftp/vmlinux to 10.31.81.4
```

Reply:
Me and Joe decided that this was the best way to check the media for the RPI. Originally, I used the method on line 28 to check for that string in the `dnsmasq` test, but it made more sense to just check if the service is running. However, the method is still a viable way to check that the RPI media is solid.

Reply:
Ah I see, the test is meant to check the media for the RPi, that makes sense. Just a little confused by the error message for the test since it mentions dnsmasq sending files to the RPi.